PageViews/PageViews.module.less

```diff
@@ -14,6 +14,10 @@
 		justify-content: center;
 	}
 
+	&.number .navButton {
+		width: 108px;
```

Comment:
Could you please define these values to `variables.less`?

Reply:
One more thing.
Please change the indentation of `.navButton {` at line 9 to a tab instead of 4 spaces.

Reply:
It's fixed.